frontend/src/scenes/session-recordings/player/sessionRecordingPlayerLogic.ts

```diff
@@ -172,6 +172,22 @@ export const sessionRecordingPlayerLogic = kea<sessionRecordingPlayerLogicType>(
                 setCurrentTimestamp: (_, { timestamp }) => timestamp,
             },
         ],
+        timestampChangeTracking: [
```

Comment:
there are a bunch of spelling changes in this PR  to help reduce noise in my IDE 🙈 

But this is a real change, we already set the timestamp every time it changes so a reducer can track whether the timestamp has stopped changing